client/extensions/woocommerce/app/order/order-payment/index.js

```diff
@@ -76,7 +83,8 @@ class OrderPaymentCard extends Component {
 
 	getPaymentAction = () => {
 		const { order, translate } = this.props;
-		if ( 'refunded' === order.status ) {
+		const codProcessing = 'cod' === order.payment_method && 'processing' === order.status;
```

Comment:
Duplicate logic from line 86, maybe we should extract it out - or just leave it - 6 one way, half dozen the other 😆

Reply:
Had the same thought, but then I'd have to write tests for another status function 😆